### PR TITLE
add tooltip component

### PR DESCRIPTION
**Pull request by magsyg**
Add a tooltip componennt, can display both text and images

Useful for showing items related to an object without displaying all images

bilde

bilde

### CODE REVIEW

File: frontend/src/Components/ToolTip/ToolTip.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  value?: string;
+  children: Children;
+};
+
+export function ToolTip({ value = 'Jokes on you', display = 'text', alignment = 'top', children }: ToolTipProps) {
+  const [hover, setHover] = useState<boolean>(false);
+
+  useEffect(() => {
+    console.log(hover);
+  }, [hover]);
```
Comment (emilte) [inline frontend/src/Components/ToolTip/ToolTip.tsx:21]: .

File: frontend/src/Components/ToolTip/ToolTip.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  return (
+    <div>
+      <div style={{ position: 'relative' }} onMouseEnter={() => setHover(true)} onMouseLeave={() => setHover(false)}>
+        {children}
+        <div
+          className={classNames(styles.tooltip, alignment == 'top' ? styles.top : styles.left, !hover && styles.hidden)}
+        >
+          {display == 'text' ? <p>{value}</p> : <Image className={styles.noMargin} src={value} />}
```
Comment (emilte) [inline frontend/src/Components/ToolTip/ToolTip.tsx:29]: Generelt aldri bruk dobbel ==

Comment (emilte) [inline frontend/src/Components/ToolTip/ToolTip.tsx:29]:
```suggestion
          {display === 'text' ? <p>{value}</p> : <Image className={styles.noMargin} src={value} />}
```

**Review by emilte — APPROVED** [review]

**Review by Snorre98 — APPROVED** [review]

File: frontend/src/Components/ToolTip/ToolTip.tsx (hunk 1 above)
Comment (magsyg) [reply frontend/src/Components/ToolTip/ToolTip.tsx:21]: Done

File: frontend/src/Components/ToolTip/ToolTip.tsx (hunk 2 above)
Comment (magsyg) [reply frontend/src/Components/ToolTip/ToolTip.tsx]: Done

Comment (magsyg) [reply frontend/src/Components/ToolTip/ToolTip.tsx]: Done

**Pull request merged**